packages/web-api/get-report-func/function.json

```diff
@@ -5,7 +5,7 @@
             "name": "request",
             "type": "httpTrigger",
             "direction": "in",
-            "route": "scans/{scanId:guid}/reports/{reportId:guid}",
+            "route": "scans/{scanId}/reports/{reportId}",
```

Comment:
what was the experience before & after this change?
how does azure function handle if scan id was not guid before?
if we make this change, we need to do scan id & report id type check in code. are we doing this?

Reply:
the code already checks if the guids are valid, but if we add the guid type in the http route then the function app is only triggered if the id is formatted like a guid. So right now, we have the guid check in code, but that code only runs if we get a valid guid in the first place. After the change, the function app will get triggered whatever the scanId is, and then we can check the scanId and send an error message if the check fails.